bot/exts/moderation/infraction/_utils.py

```diff
@@ -94,6 +95,14 @@ async def post_infraction(
 
     current_time = arrow.utcnow()
 
+    if any(
+        is_in_category(ctx.channel, category)
+        for category in (Categories.modmail, Categories.appeals, Categories.appeals2)
+    ):
+        jump_url = "Infraction issued in a ModMail channel."
```

Comment:
Either something has a jump link or it doesn't, storing a string in here too means we can't have a validator on the db that it's a valid link.

IMO we should have a validator on site that requires it to be formatted as a link, or null.

Reply:
I could be misremembering but Ive read somewhere (django docs or stackoverflow) that using blank strings is recommended by django ORM. I can look for the link later.

Reply:
Maybe it's just a convention.

Reply:
And regarding the validator, I could use URLField instead of CharField because it has URL validation.

Reply:
A URLField that can be blank/null sounds good to me